AndelaOSP/art-backend
# create history endpoint

#### What does this PR do?
Adds a history endpoint to the application

#### Description of Task to be completed?
This endpoint enables an admin to track create, update and delete actions done in the application

#### How should this be manually tested?
1. Make a create, update or delete action and after navigate to the `http://127.0.0.1:8000/api/v1/history` history endpoint and make a get request. You will see all actions performed on the application.


#### Any background context you want to provide?
after pulling the branch, Run migrations to have the history model in your database.


#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2146417/stories/167008828
